README.md

```diff
@@ -88,13 +79,17 @@ It should look something like this:
 
```

Comment:
Is the location of the `apps.json` still in that subdirectory, or also moved as `dindctl` is moved?

Reply:
Yes, it's the same